src/test/java/RunCucumberTest.java

```diff
@@ -8,10 +8,13 @@
 import java.util.Properties;
 
 @RunWith(Cucumber.class)
-@CucumberOptions(plugin = {"pretty", "html:target/cucumber-reports" },
-                 glue = {"odysseusAT"}
-//                , tags = {"@dev"}
-                )
+@CucumberOptions
+
+        (plugin = {"pretty", "json:target/cucumber.json"},
+                glue = {"atlastests"}
+                , tags = {"not @ds and not @qds"}
+//                    , tags = {"@1209"}
```

Comment:
Why is it still here?

Reply:
Because it is configuration, I think I shouldn't remove it.